### PR TITLE
Irina og3 16 exit gracefully

### PR DESCRIPTION
**Pull request by khafizka**
In this PR you may find:
1.flexible ports: default is set to 6661 and 6662 and will be run with command: npm run dev for now
dynamic ports: any port you want for example: PORT1=4041 PORT2=4042 npm run dev  (dev for now)
2. Exit /shutdown closes all open windows after the docker shutdown

### CODE REVIEW

File: src/pages/api/kite/shutdown.ts [hunk 1]
```diff
@@ -13,6 +20,15 @@ export default async function handler(
...
     const kite = Kite.shutdown();
+    for (const url of urls) {
+      if (process.platform === 'darwin') {
+        // For macOS
+        spawn('osascript', ['-e', `tell application "Google Chrome" to close (tabs of window 1 whose URL is "${url}")`]);
+      } else {
+        // For other platforms
+        spawn('taskkill', ['/F', '/IM', 'chrome.exe']);
+      }
+    }
```
Comment (zahara-aviv) [inline src/pages/api/kite/shutdown.ts:30]: This works for chrome, but not for all browsers. Good for MVP, but we may want to research a platform independent way. Perhaps redirect to landing page would be easiest / consistent?

Comment (zahara-aviv) [reply src/pages/api/kite/shutdown.ts:30]: Also what about me and my linux machine use case :( lol

Comment (khafizka) [reply src/pages/api/kite/shutdown.ts:30]: That's weird I changes a command to accommodate all browsers, and it's not reflected in this PR, oone moment I will take a look at why this shows an old version of the file

Comment (khafizka) [reply src/pages/api/kite/shutdown.ts:38]: Found it in my last stash:( This accounts for Linux and theoretically should work, but if it gives you a hard time, please let me know, I will work on it:)

**Review by zahara-aviv — APPROVED** [review]

**Pull request merged**